src/services_models/reserve_group.jl

```diff
@@ -60,10 +60,12 @@ function add_constraints!(
     for t in time_steps
         resource_expression = JuMP.GenericAffExpr{Float64, JuMP.VariableRef}()
         for reserve_variable in reserve_variables
-            JuMP.add_to_expression!(resource_expression, sum(reserve_variable[:, t]))
+            JuMP.add_to_expression!(resource_expression,
```

Comment:
Is this more effective than pre-allocating the expression in 61?

Reply:
not sure I got what you mean